Content.Client/StationRecords/GeneralStationRecordConsoleWindow.xaml.cs

```diff
@@ -119,6 +119,10 @@ private void PopulateRecordContainer(GeneralStationRecord record)
             new Label()
             {
                 Text = Loc.GetString("general-station-record-console-record-fingerprint", ("fingerprint", record.Fingerprint is null ? Loc.GetString("generic-not-available-shorthand") : record.Fingerprint))
+            },
+            new Label()
+            {
+            Text = Loc.GetString("general-station-record-console-record-dna", ("dna", record.DNA is null ? Loc.GetString("generic-not-available-shorthand") : record.DNA))
```

Comment:
> ?? for the thing is null ? : thing bit

wdym?

Reply:
`record.DNA ?? Loc.GetString("generic-not-available-shorthand")`
should work i think

Reply:
might be worth moving the string to the component, simply cause i have had to do this in previous prs to satisfy the maintainers and might save some time rather than possibly having to do it later.